lib/python/picongpu/pypicongpu/species/species.py

```diff
@@ -75,7 +95,7 @@ def check(self) -> None:
         # position
         if Position not in [type(a) for a in self.attributes]:
             raise ValueError("Each species must have the position attribute!")
-        # momentum
+        # momentum, @todo really necessary?, Brian Marre, 2024
```

Comment:
This is a requirement set by PyPIConGPU currently.
PIConGPU does not care and delegates responsibility to the user for choosing the correct attributes.

In theory it would be imaginable have species without a momentum, i.e. un-moving probe-particles, but that is not currently supported by PyPIConGPU because Hannes decided to make the momentum attribute mandatory as there is now way to specify a species as unmoving in the PICMI standard.